src/jesse_state.erl

```diff
@@ -295,16 +295,11 @@ load_local_schema(Schema, [Key | Keys]) ->
 combine_id(Id, undefined) ->
   Id;
 combine_id(Id, RefBin) ->
-  Ref = unicode:characters_to_list(RefBin),
-  case http_uri:parse(Ref) of
-    %% Absolute http/s:
-    {ok, _} ->
-      Ref;
-    %% Absolute file:
-    {error, {no_default_port, file, Ref}} ->
+  case parse_ref(RefBin) of
+    {absolute, Ref} ->
       Ref;
     %% Relative
```

Comment:
remove the `%% Relative` comment. it is now nicely readable in the returned tuple 👏